addons/l10n_ch/__init__.py

```diff
@@ -14,8 +14,8 @@ def load_translations(env):
 def init_settings(env):
     '''If the company is localized in Switzerland, activate the cash rounding by default.
     '''
-    # The cash rounding is activated by default only if the company is localized in Switzerland.
-    for company in env['res.company'].search([('partner_id.country_id.code', '=', "CH")]):
+    # The cash rounding is activated by default only if the company is localized in Switzerland or Liechtenstein.
```

Comment:
I'm not sure about this. Is it really related to the QR Bills generation? Is the legislation the same on this specific point?

Reply:
taking this by the word:

> @momegahed I had the change to check this with @jco-odoo during the Odoo XP

👍 for this PR which solves generation of invoice from CH company to LI customer.

(thanks for the pings)

ℹ️ On a more general note, you will want to consider Liechtenstein as a Switzerland region as it uses the same norms and specs.
So almost anytime you have a if ... = "CH" you will want to also make the same condition for "LI". BTW the QR-Bill specs clearly mention LI. (For instance you can use LI IBAN)

I understand that they share exactly the same accounting standards.